tests/js/HappinessLintBearTest.py

```diff
@@ -17,3 +23,19 @@
 HappinessLintBearTest = verify_local_bear(HappinessLintBear,
                                           valid_files=(good_file,),
                                           invalid_files=(bad_file,))
+
+
+@generate_skip_decorator(HappinessLintBear)
+class HappinessLintBearConfigTest(LocalBearTestHelper):
+
+    VALUE_ERR_RE = ('Set "use_spaces=False" for HappinessLintBear '
+                    'as "use_spaces=Ture" is incompatible '
+                    'for HappinessLintBear.')
```

Comment:
use `re.escape` on it, otherwise special characters like `.` may have side effects.